src/test/miner_tests.cpp

```diff
@@ -89,8 +89,10 @@ static CBlockIndex CreateBlockIndex(int nHeight, CBlockIndex* active_chain_tip)
 // Test suite for ancestor feerate transaction selection.
 // Implemented as an additional function, rather than a separate test case,
 // to allow reusing the blockchain created in CreateNewBlock_validity.
-void MinerTestingSetup::TestPackageSelection(const CChainParams& chainparams, const CScript& scriptPubKey, const std::vector<CTransactionRef>& txFirst)
+void MinerTestingSetup::TestPackageSelection(const CScript& scriptPubKey, const std::vector<CTransactionRef>& txFirst)
 {
+    CTxMemPool tx_mempool{/*estimator=*/nullptr, /*check_ratio=*/1};
```

Comment:
I think this conflicts with @dongcarl 's intent to construct less mempools (although maybe this isn't a problem as this is instantiated with special parameters).

Reply:
Good point. Fixed by constructing a mempool in only one place